src/Analysim.Web/ClientApp/src/app/projects/delete-project/delete-project.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Isn't this component not used? There's another `projects/project/modal-delete/modal-delete.component.*` below.

If so, delete all of the component files @kiyros